src/models/stackedAreaChart.js

```diff
@@ -367,6 +367,8 @@ nv.models.stackedAreaChart = function() {
                             color: color(series,series.seriesIndex),
                             stackedValue: point.display
                         });
+
+                        valueSum += tooltipValue;
```

Comment:
Hmm well it's computed with
```
var tooltipValue = (stacked.style() == 'expand') ? point.display.y : chart.y()(point,pointIndex);
```
So if we're not in expanded mode, I'm pretty confident we're getting a number. Not sure about expanded mode, but it doesn't matter, since we don't actually add a TOTAL row in that case. To make sure I'm not doing anything silly like concatenating a bunch of strings, I just added an `if` to protect this.